pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [tool.poetry]
-name = "stacks-azure-data-platform"
+name = "datastacks"
```

Comment:
If I understand it correctly, if we want it to be a proper Python library, that we can build and generate a wheel, then I think the tests folder will have to stay at the top level, meaning:

```
stacks-azure-data
├── pyproject.toml
├── datastacks
│   └── __init__.py
└── tests
    └── __init__.py
```

That means having `tests` folder lost between different other stuff. Are we okay with it?

Reply:
I don't neccessarily like it, but if that's what needs to be done then that's fine

Reply:
OK, I don't mind it, just wanted to point that out:) Resolving.